### PR TITLE
Feat/colors extensions

### LINKED ISSUE #1425: Color helpers
Should be added as Optional Service
Depends on Colors Service

Usage example:
```html
<span class="text--primary bg--secondary">primary secondary</span>
```

We had this functionality before, but it disappeared mysteriously 🤪.
- [ ] Service should generate CSS dynamically based on colors config. I suggest you look into prior implementation (couple years back should be enough). The idea is that we attach `styles` to `head`, then replace that block on colors change. Old implementation was well implemented.
- [ ] should support only text and background: `text--primary`, `bg--secondary`
- [ ] Should have no configuration
- [ ] CSS helpers should rely on CSS variables (that are part of Colors Service).
- [ ] Document Color helpers as a separate page in Services category. Add installation instructions.
- [ ] There are still some places with `text--primary`, these do nothing and we need to fix these (remove or make do something).

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Close #1425

image

## Description
- Added UnoCSS integration guide page.
- Translations.
- Setup example.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/page-config/extensions/unocss/code/unocss.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  presets: [
+    presetUno({
+      dark: 'media',
+    }),
+    presetIcons(),
+    presetTheme({
+      theme: {
+        dark: {
+          colors: colorsPreset.dark,
+        },
```
Comment (m0ksem) [inline packages/docs/page-config/extensions/unocss/code/unocss.config.js:26]:
```suggestion
        dark: {
          colors: colorsPreset.dark,
        },
        light: {
           colors: colorsPreset.light
        }
```

Seems quite logical to me. Not sure if it is correct.

File: packages/docs/page-config/extensions/unocss/code/unocss.config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default defineConfig({
+  theme: {
+    colors: colorsPreset.light,
+    breakpoints: convertNumbersToCssUnits(thresholdsPreset),
+  },
+  presets: [
+    presetUno({
+      dark: 'media',
+    }),
+    presetIcons(),
```
Comment (m0ksem) [inline packages/docs/page-config/extensions/unocss/code/unocss.config.js:21]: Is this required?

**Review by m0ksem — APPROVED** [review]

File: packages/docs/page-config/extensions/unocss/code/unocss.config.js (hunk 1 above)
Comment (rustem-nasyrov) [reply packages/docs/page-config/extensions/unocss/code/unocss.config.js:26]: As I understood, preset was designed like that.

**Review by aluarius — APPROVED** [review]

**Comment by m0ksem** [comment]
Can you resolve this conflict? Try to use split translations here.

**Pull request merged**